packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -147,16 +148,23 @@ export const LoadedEditor: React.FC<LoadedEditorProps> = ({
   }, [webLinter, onEsASTChange, onScopeChange, onTsASTChange]);
 
   useEffect(() => {
+    const createRuleUri = (name: string): string =>
+      monaco.Uri.parse(`/rules/${name.replace('@', '')}.json`).toString();
+
     // configure the JSON language support with schemas and schema associations
     monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
       validate: true,
       enableSchemaRequest: false,
       allowComments: true,
       schemas: [
+        ...Array.from(webLinter.rules.values()).map(rule => ({
+          uri: createRuleUri(rule.name),
+          schema: getRuleJsonSchemaWithErrorLevel(rule.name, rule.schema),
+        })),
```

Comment:
[Bug] I really like this in general, but there's a `: ` in front of the complaint. And it's listed under _**json**_, which isn't very specific.

![Screenshot of the complaint listing under 'json' starting with a colon](https://user-images.githubusercontent.com/3335181/232329971-b367d83c-45af-4d6d-bd72-d60fc21c4399.png)

Can we put something in front of the `:` at least?

Reply:
hmm error handling/displaying, this is actually not related to this PR, but rater to how error viewer is handling monaco errors, in case of TS, we are receiving TS error, and there is no check if code is actually present

i will try to improve it a little